### DISCUSSION TITLE
Batch Effects in RNA-Seq

### DISCUSSION THREAD

**Discussion by josesanchez1815** [nih-cfde/training-and-engagement#452] [RNA - Seq] [answered]
Is there a difference between blocking effect and batch effect?
Does Blocking effect speak to the 'power' or Type I//II error of an experiment?

**Comment by ACharbonneau** [comment, marked as answer]
Blocking and batch are usually synonyms. They're any thing that happens to your experiment in a reasonably stable, grouped way. So, if you did your experimental treatment over two days, day might have an effect. If you have two different people take measurements, person could be an effect. Technically speaking, your experimental treatment is also a batch or block effect, it just is one you care about. Usually when we say blocking or batch we're implying that it's not the experimental treatment though. 

Batch effects can cause any kind of error, depending on how strong they are and what they are correlated with. If you don't account for them, and they have a strong effect, and they are correlated with your variable of interest, they could make it seem like you have an effect when you don't. Or, if they have a strong effect opposite your variable of interest they could drown out a real signal in your data.
